File: src/sdk-reference/csharp/1/document/create/snippets/create.cs
try {
  string response;
  response = kuzzle.document.create(
    "nyc-open-data",
    "yellow-taxi",
    "some-id",
    @"{""lastName"": ""Eggins""}");

  Console.WriteLine(response);
  /*
  {
    "_index": "nyc-open-data",
    "_type": "yellow-taxi",
    "_id": "some-id",
    "_version": 1,
    "result": "created",
    "_shards": {
      "total": 2,
      "successful": 1,
      "failed": 0
    },
    "created": true,
    "_source": {
      "lastName": "Eggins",
      "_kuzzle_info": {
        "author": "-1",
        "createdAt": 1537445737667,
        "updatedAt": null,
        "updater": null,
        "active": true,
        "deletedAt": null
      }
    }
  }
  */

  Console.WriteLine("Document successfully created");
} catch (KuzzleException e) {
  Console.Error.WriteLine(e.getMessage());
}

File: src/guide/1/dbsearch/javascript/snippets/create.js
try {
  // Delete the nyc-open-data index if exists
  if (await kuzzle.index.exists('nyc-open-data')) {
    await kuzzle.index.delete('nyc-open-data');
  }

  // Create a "nyc-open-data" index, a "yellow-taxi" collection
  // and 2 documents
  // with different licence property values
  await kuzzle.index.create('nyc-open-data');
  await kuzzle.collection.create('nyc-open-data', 'yellow-taxi');
  await kuzzle.document.create(
    'nyc-open-data',
    'yellow-taxi',
    { licence: 'B' }
  );
  await kuzzle.document.create(
    'nyc-open-data',
    'yellow-taxi',
    { licence: 'C' }
  );

  console.log('New documents successfully created!');
} catch (error) {
  console.error(error);
}
